.travis.yml

```diff
@@ -2,5 +2,8 @@ language: python
 python:
   - 3.6
 cache: pip
+install:
+  - pip3 install pipenv
+  - pipenv install
 script:
-  - python3 robot.py test
\ No newline at end of file
+  - python3 robot.py test
```

Comment:
I believe this line should be `pipenv shell python robot.py test`. At the moment the `cache: pip` is keeping the deps available through regular pip, but we want to run in the `pipenv shell` because that automatically gets updated everytime we do a `pipenv install` if necessary

Reply:
That would be my understanding as well.

Reply:
In that case, should `cache: pip` be removed? Also, is a matching `exit` needed after `pipenv shell`?

Reply:
good catch re:exiting, `pipenv shell "python robot.py test && exit"` seems to work for me, but please do a test where you deliberately inject a failing test and make sure travis detects and shows the failure

Reply:
`cache: pip` will make sure pipenv stays installed, which is fine

Reply:
I tried a failing test (locally) and the exit status of the pipenv command was not zero, so it should be good.

Reply:
Great work. This is good to go